src/borg/testsuite/archiver.py

```diff
@@ -1132,6 +1132,17 @@ def test_list_prefix(self):
         self.assert_in('test-2', output)
         self.assert_not_in('something-else', output)
 
+    def test_list_repository_format(self):
+        self.cmd('init', self.repository_location)
+        self.cmd('create', self.repository_location + '::test-1', src_dir)
+        self.cmd('create', self.repository_location + '::test-2', src_dir)
+        output_1 = self.cmd('list', self.repository_location)
+        output_2 = self.cmd('list', '--format', '{name:<36} {time} [{id}]{NL}', self.repository_location)
+        self.assertEqual(output_1, output_2)
+        output_1 = self.cmd('list', '--short', self.repository_location)
+        output_2 = self.cmd('list', '--format', '{name}{NL}', self.repository_location)
+        self.assertEqual(output_1, output_2)
+
```

Comment:
if you have done the above requested changes (using an ArchiveFormattter), some of these tests get pointless and need to actually check whether the output looks as expected (not just test for equality with some other output generated in the same way).